README.md

```diff
@@ -1,454 +1,416 @@
-# Gamepad mapping
-WebXR hardware may support a variety of input sources including motion controllers. When a motion controller is present, developers often wish to do the following:
+# Gamepad Mapping
+
+## Motivation
+> **TODO** Explain what WebXR is and how its different from WebVR
+
+> **TODO** Define motion controllers
+
+When a motion controller is present on XR hardware, developers often wish to do the following:
 1. Display the motion controller's virtual model at the correct location
-1. Find out the state of the controller's buttons and axes
-1. Modify the virtual model to reflect the state of the buttons and axes each render frame
-1. Provide a key or legend that explains the mapping of button/axes to actions
+1. Find out the state of the controller's component parts (made up of buttons and axes)
+1. Modify the virtual model to reflect the state of the components each render frame
+1. Provide a key or legend that explains the mapping of components to actions
+
+The state of an XR motion controller's buttons, thumbsticks, dpads, and touchpads is made available to developers via the the [Gamepad API](https://www.w3.org/TR/gamepad/). This data is divided up and populated in the `Gamepad.buttons` array and the `Gamepad.axes` array. While this system was adequate for the relatively homogenous console gaming controllers, it breaks down for XR motion controllers as they have not yet converged on a common form factor. In addition, the [Gamepad API](https://www.w3.org/TR/gamepad/) does not provide any information about the visualization of a `Gamepad` object which is a requirement to displaying a virtual copy of motion controller on opaque XR headsets.
```

Comment:
> The state... is made available to developers via the the Gamepad API.

Nit: That seems to suggest that it would be accessible via `navigator.getGamepads`. I suggest instead "A `Gamepad` object, as defined by the Gamepad API"